diktat-rules/src/test/resources/test/paragraph3/long_line/LongLineCommentExpected2.kt

```diff
@@ -0,0 +1,22 @@
+package test.paragraph3.long_line
+
+/**
+ * This rule checks if there is a backing property for field with property accessors, in case they don't use field keyword
+ */
+class ImplicitBackingPropertyRuleTest(configRules: List<RulesConfig>) : DiktatRule(
+    "implicit-backing-property",
+    configRules,
+    listOf(NO_CORRESPONDING_PROPERTY)) {
+    override fun logic(node: ASTNode) {
+        if (node.elementType == CLASS_BODY) {
+            findAllProperties(node)
+        }
+    }
+
+    private fun validateAccessors(node: ASTNode, propsWithBackSymbol: List<String>) {
+        val accessors = node.findAllDescendantsWithSpecificType(PROPERTY_ACCESSOR).filter { it.hasChildOfType(BLOCK) }  // exclude get with expression body
```

Comment:
Is this the line that caused an error? Could you make this example more minimal, e.g. the `logic` function doesn't have anything to do with line length here.

Reply:
Thanks, agree, now this test is more compact